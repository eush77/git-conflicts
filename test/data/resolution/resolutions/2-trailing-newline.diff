<<<<<<< 2b0b56a7b74d4f5a0a7ef042c34eb27981ab9859
Can optionally be newline-terminated.
>>>>>>> Refactor
