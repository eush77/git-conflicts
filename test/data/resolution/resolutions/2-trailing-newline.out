Can optionally be newline-terminated.